frontend/src/components/homepage/components/search/common/types/podcast-search-payload.type.ts

```diff
@@ -0,0 +1,7 @@
+import { PodcastSearchPayloadKey } from '../enums/search';
```

Comment:
I think it should be at the level above

Reply:
I checked it, its fine

Reply:
I mean it should be here
`frontend/src/components/homepage/common/types/podcast-search-payload.type.ts`

Reply:
do we use it somewhere?